analysis/val02_counters/counters01_prepare_data_nb.py

```diff
@@ -58,8 +67,28 @@
 BERLIN_PATH = DATA_PATH / 'loop_counters' / 'berlin'
```

Comment:
Do we want use `DATA_PATH = Path ('/iarai/public/t4c/mets10/')` to have same paths locally and on the fs?